src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.spec.ts

```diff
@@ -469,6 +469,18 @@ describe('HearingEditSummaryComponent', () => {
     expect(hearingsService.propertiesUpdatedOnPageVisit.afterPageVisit.partyDetailsChangesRequired).toEqual(true);
   });
 
+  it('should partyDetailsChangesRequired return true if party type changed', () => {
+    spyOn(store, 'select').and.returnValue(of(initialState.hearings.hearingValues));
+    component.serviceHearingValuesModel = _.cloneDeep(initialState.hearings.hearingValues.serviceHearingValuesModel);
+    component.serviceHearingValuesModel = {
+      ...component.serviceHearingValuesModel,
+      parties: initialState.hearings.hearingRequest.hearingRequestMainModel.partyDetails
+    };
+    component.serviceHearingValuesModel.parties[0].partyType = PartyType.ORG;
```

Comment:
@johnbenjamin-hmcts - is it guaranteed there will always be at least one item in `parties` array?